### ISSUE TITLE
Respinning ...

### ISSUE THREAD

**Issue by Plasmoner** [Plasmoner/If-I-break-something-I-get-to-keep-both-halfs-#1]
My first attempt to respin Kubuntu 16.04.3 in order to create an .iso that has the 5.8.x backport included.
***
![screenshot_20171104_211154](https://user-images.githubusercontent.com/29124716/32409060-dd6196aa-c1ac-11e7-92ce-b821130d8a08.png)
***
Q1: `What exactly do I type in for the backport when I edit the sources`
***
Help wanted and appreciated. Insights, advices, comments, recommendations, etc, are more than welcome.

Note: this is the tool I've got in place https://github.com/kamilion/customizer

**Comment by Plasmoner** [comment]
Oh, forgot to mention ... this is the host
***
![screenshot_20171105_001158](https://user-images.githubusercontent.com/29124716/32409910-5dfab7cc-c1be-11e7-9226-599d123700e7.png)
***
If you like Ubuntu and Plasma, then this is the silkiest experience to date.

**Comment by Plasmoner** [comment]
Project update no. 1
***
I managed to pull it off.
After about 30 seconds worth of digging and tinkering, I performed:
`root@hostname:/# apt update`
...
`root@hostname:/# apt full-upgrade`
...
`root@hostname:/# add-apt-repository ppa:kubuntu-ppa/backports`
...
`root@hostname:/# apt update`
...
`root@hostname:/# apt full-upgrade`
...
Then I clicked on the "Rebuild ISO" button in the GUI. 
(I should have used `-r, --rebuild`)
...
Waited ...
...
Burnt newly created image onto a DVD with K3b.
...
Installed from the install disk. The whole installation went perfectly well. Actually, unexpectedly well.
***
To sum it up, I achieved my purpose.
***
To do: repeat the process several times until it becomes an optimal routine.
***
![screenshot_20171105_130317](https://user-images.githubusercontent.com/29124716/32414172-db6f89c4-c229-11e7-96f7-568b765c81e3.png)
